### PR TITLE
misc: Add install libunwind lib and WSL2 Fedora support in install-deps.sh

### PR DESCRIPTION
**Pull request by paranlee**
Add install libunwind library and WSL2 Fedora support in issue #1515

Signed-off-by: Paran Lee <p4ranlee@gmail.com>

### CODE REVIEW

**Comment by honggyukim** [comment]
Thanks @paranlee, but please have a look at #1382.

**Comment by paranlee** [comment]
@honggyukim , Thanks you for review! 

Would it be better to create a separate issue for WSL2 fedora support only?

**Comment by honggyukim** [comment]
> WSL2 fedora

What do you mean by WSL2 fedora support? Isn't it same as normal fedora?

**Comment by paranlee** [comment]
@honggyukim Updated PR. 

When DEBUG=1, because libunwind-dev is only required in debug build.

` [ $DEBUG ] && apt-get $OPT install libunwind-dev `

#1515 : There are additional postfix on ID=fedoraremixforwsl

[kim@DESKTOP-S4LJL03 uftrace]$ cat /etc/os-release
NAME="Fedora Remix for WSL"
VERSION="36"
ID=fedoraremixforwsl
ID_LIKE=fedora
VERSION_ID=36
PLATFORM_ID="platform:f36"

File: misc/install-deps.sh [hunk 1]
```diff
@@ -21,23 +21,28 @@ case $distro in
     "ubuntu" | "debian" | "raspbian" | "kali" | "linuxmint")
         apt-get $OPT install pandoc libdw-dev python3-dev libncursesw5-dev pkg-config
         apt-get $OPT install libluajit-5.1-dev || true
+        [ $DEBUG ] && apt-get $OPT install libunwind-dev || true
         apt-get $OPT install libcapstone-dev || true ;;
-    "fedora")
+    "fedora"*)
```
Comment (namhyung) [inline misc/install-deps.sh:26]: No wildcard plesae.  I'd like to fix it with ID_LIKE as in #1468.

**Review by namhyung — COMMENTED** [review]
I don't have a strong opinion on the libunwind installment. @honggyukim ?

File: misc/install-deps.sh [hunk 2]
```diff
@@ -21,23 +21,28 @@ case $distro in
     "ubuntu" | "debian" | "raspbian" | "kali" | "linuxmint")
         apt-get $OPT install pandoc libdw-dev python3-dev libncursesw5-dev pkg-config
         apt-get $OPT install libluajit-5.1-dev || true
+        [ $DEBUG ] && apt-get $OPT install libunwind-dev || true
```
Comment (honggyukim) [inline misc/install-deps.sh:24]: We use `DEBUG=1` only during `make` stage as of now. I think it's better to provide a way to install libunwind as follows.
```
$ ./misc/install-deps.sh libunwind
```

**Review by honggyukim — COMMENTED** [review]
It looks this PR does libunwind installation and WSL2 Fedora support, but I think they have to be in separate patches.

File: misc/install-deps.sh (hunk 2 above)
Comment (paranlee) [reply misc/install-deps.sh:24]: Nice! :)
I'm going to isolate the issue.

File: misc/install-deps.sh (hunk 1 above)
Comment (paranlee) [reply misc/install-deps.sh:26]: Thanks you for code review. :)

**Pull request closed without merging**